src/core/load-fragment.c

```diff
@@ -3835,16 +3836,18 @@ int config_parse_managed_oom_mode(
 
         if (isempty(rvalue)) {
                 *mode = MANAGED_OOM_AUTO;
-                return 0;
+                goto finish;
         }
 
         m = managed_oom_mode_from_string(rvalue);
         if (m < 0) {
                 log_syntax(unit, LOG_WARNING, filename, line, 0, "Invalid syntax, ignoring: %s", rvalue);
                 return 0;
         }
-
         *mode = m;
+
+finish:
+        (void) manager_varlink_send_managed_oom_update(u);
```

Comment:
this doesn't look right. We should not send out any change just because we parsed something, after all there might be  further settings in the same unit file that turn this off again.

move this to unit_load(), after the unit was fully loaded, i.e. where we also at the unit to the dbus queue using unit_add_to_dbus_queue()